src/cephadm/cephadm.py

```diff
@@ -3963,6 +4111,8 @@ def _write(conf: Path, lines: List[str]) -> None:
         lines = HAproxy.get_sysctl_settings()
     elif daemon_type == 'keepalived':
         lines = Keepalived.get_sysctl_settings()
+    elif daemon_type == 'nvmeof':
```

Comment:
```suggestion
    elif daemon_type == CephNvmeof.daemon_type:
```
just for consistency

Reply:
Fixed